packages/valory/services/oracle_goerli/service.yaml

```diff
@@ -99,7 +123,15 @@ type: skill
         url: https://api3.drand.sh/public/latest
         api_id: protocollabs2
     params:
-      args: *id001
+      args:
+        setup: *id001
+        max_healthcheck: 43200
+        tendermint_p2p_url: ${TM_P2P_ENDPOINT_NODE_3:str:node3:26686}
```

Comment:
Would be good to have this set automatically, like we do for the other tendermint urls.